src/test/groovy/net/fortuna/ical4j/data/CalendarBuilderSpec.groovy

```diff
@@ -0,0 +1,67 @@
+package net.fortuna.ical4j.data
+
+import net.fortuna.ical4j.model.Calendar
+import net.fortuna.ical4j.util.CompatibilityHints
+import spock.lang.Specification
+
+import java.nio.charset.Charset
+
+/**
+ * Created by fortuna on 4/07/2016.
+ */
+class CalendarBuilderSpec extends Specification {
+
+    def 'test relaxed parsing'() {
+        given: 'a calendar object string'
+        String cal2 = '''BEGIN:VCALENDAR
```

Comment:
This should preferably be put into a test resource file. With the correct encoding and line breaks.